services/121-service/src/programs/program/program.controller.spec.ts

```diff
@@ -35,6 +35,7 @@ const newProgramParameters = {
   highestScoresX: 500,
   meetingDocuments: JSON.parse('{"en": "documents"}'),
   customCriteria: [],
+  notifications: JSON.parse('{}')
```

Comment:
Why all the `JSON.parse`-uses?

That just literally 'removes the quotes'? Why not create the whole object as a real object?